packages/components/src/components/post-header/post-header.scss

```diff
@@ -42,6 +43,11 @@
   .local-sub {
     display: none;
   }
+  .navigation {
```

Comment:
This does not seem to be applied:
![image](https://github.com/user-attachments/assets/32bcb6dd-ff9b-4331-8e8b-84cf171a4b2c)